app/views/content_pages/interruption_page.html.slim

```diff
@@ -4,11 +4,8 @@
   .interrupt-panel.govuk-grid-column-full
     h1 What to expect during the training
     .gem-c-govspeak
-      - if @model.formative?
-        = translate_markdown t('interruption.formative')
-      - else
-        = translate_markdown t('interruption.summative')
-
+      = translate_markdown t('interruption.formative') if @model.formative
+      = translate_markdown t('interruption.summative') if @model.summative
```

Comment:
@peterdavidhamilton Laura specifically asked for this section to only show if there are formative questions (I realize the `?` is missing here as I type) which is why I kept it.  Can you have a chat with her - I think she did say this, but as all modules have formative questions the paragraph will always show....